test/end-to-end/entrypoint.sh

```diff
@@ -8,7 +8,7 @@ sed -i "s,\"uri\": \"http://localhost:4000/\",\"uri\": \"http://api:4000/\"," co
 if [ -z "$COCKPIT_TEST" ]; then
     sed -i "s,\"root\": \"http://localhost:3000/\",\"root\": \"http://web:3000/\"," config.json
 else
-    sed -i "s,\"root\": \"http://localhost:3000/\",\"root\": \"http://web:9090/welder\"," config.json
+    sed -i "s,\"root\": \"http://localhost:3000/\",\"root\": \"http://web:9090/welder-web\"," config.json
```

Comment:
Why is this change necessary ?

Reply:
The RPM package installed into /usr/share/cockpit/welder-web directory, so the URL should be changed.

Reply:
ok then

Reply:
Still,  this URL hack is rather ugly. You could instead add a `"name": "welder"` into the cockpit manifest, or better yet, harmonize the install path and the name to all be `welder`?

Reply:
@jgiardino and @jkozol, is that possible to change RPM install path to welder? Thanks.